app/views/users/show.html.haml

```diff
@@ -0,0 +1,2 @@
+%h1 Users#show
```

Comment:
What does it mean? Will user see this text? Bad one. Please change name
%h1= 'Welcome text'